vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -94,6 +94,34 @@ public DateTimePicker() {
         this((LocalDateTime) null);
     }
 
+    /**
+     * Convenience constructor to create a date time picker with a label.
+     *
+     * @param label
+     *            the label describing the date time picker
+     * @see #setLabel(String)
+     */
+    public DateTimePicker(String label) {
+        this();
+        setLabel(label);
+    }
+
+    /**
+     * Convenience constructor to create a date time picker with a pre-selected
+     * date and time in current UI locale format and a label.
+     *
+     * @param label
+     *            the label describing the date picker
```

Comment:
"date time picker" instead of "date picker"

Reply:
Updated.